package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bytes",
   "description": "Utility to parse a string bytes to bytes and vice-versa",
-  "version": "2.4.0",
+  "version": "3.4.0",
```

Comment:
I'm not saying it is not breaking, but the release process you can see in the history is that the releaser bumps the version. The change also didn't comply with the semver spec, which requires you reset the other numbers back to zero on the major bump.